packages/salesforcedx-vscode-core/src/commands/util/libraryDeployResultParser.ts

```diff
@@ -8,21 +8,21 @@ import {
   Row,
   Table
 } from '@salesforce/salesforcedx-utils-vscode/out/src/output';
-import { nls } from '../messages';
 import {
   DeployResult,
   DeployStatusEnum,
-  ToolingRetrieveResult
-} from './deployUtil';
+  SourceResult
```

Comment:
Moved away from using the types defined in the extensions project and started using the ones shipped with the library.